include/rocksdb/options.h

```diff
@@ -1121,10 +1121,12 @@ struct DBOptions {
 
   // Use O_DIRECT for reading file
   // Default: false
+  // Not supported in ROCKSDB_LITE mode!
```

Comment:
is it too difficult to #ifndef the whole option? then we can find errors at compile-time, both for our error (we accidentally partially support use_direct_reads) and user error (user accidentally sets use_direct_reads = true on a lite build).